htdsa/client.py

```diff
@@ -59,7 +60,15 @@ def _uri(self, args=None):
 		return uri
 	
 	def _request(self, method, args, kwargs):
-		result = self.pool.request(method, self._uri(args), data=kwargs, **self.options)
+		uri = self._uri(args)
+		data = kwargs
+		if method == 'GET' and len(kwargs) > 0:
+			data = dict()
+			qs = urllib.parse.urlencode(kwargs)
+			uri += '?' if '?' not in uri else '&'
+			uri += qs
```

Comment:
Specifically, this segment can be rewritten:

```python
data = {}
uri += ('?' if '?' not in uri else '&') + urllib.parse.urlencode(kwargs)
```

However, simply passing the dict of data as the correct argument (see the Requests documentation link, above) would be preferred.